e2e/client.e2e.test.ts

```diff
@@ -402,6 +402,22 @@ describe('createVariable', () => {
   });
 });
 
+describe('listTags', () => {
```

Comment:
you should also add a test that uses the search param, see `it('should list only the workspaces that match the queried name', async () => {`